### PR TITLE
Fix AttributeError for complex.real and complex.image

### PR DESCRIPTION
**Pull request by deep110**
I have included tests for the issue https://github.com/pybee/voc/issues/378 and marked them as expected failures. I am currently unable to fix this issue. Any help regarding this would great. Thanks.

### CODE REVIEW

**Comment by eliasdorneles** [comment]
Hi @deep110 !
To fix the `AttributeError` you see after adding this, try adding `@org.python.Attribute` to the real and imag attributes in the `Complex.java` file.
You'll then get another error that's related to the `Complex.toJava()` method not being implemented, and I _think_ you can make `toJava` simply return this.
Give it a try and good luck! :)

**Comment by deep110** [comment]
Thanks @eliasdorneles. It worked like a charm. Now got a bit more understanding of internal code. :)

File: tests/datatypes/test_complex.py [hunk 1]
```diff
@@ -8,6 +8,18 @@ def test_conjugate(self):
...
+        self.assertCodeExecution("""
+            x = complex(1, 1)
+            print(x.real)
+            """)
+
+    def test_imag(self):
+        self.assertCodeExecution("""
+            x = complex(1, 2)
+            print(x.imag)
+            """)
```
Comment (eliasdorneles) [inline tests/datatypes/test_complex.py]: Could you merge these two tests into one?
They're pretty much related, and it makes the test case run slightly faster (each new test needs compiling and running code a few times).

Comment (deep110) [inline tests/datatypes/test_complex.py]: @eliasdorneles Merged the tests.

**Pull request merged**